src/components/Content.jsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react"
+import { createReviewCards } from "./utils"
+import { getReviews } from "./api";
+
+export const Content = () => {
+
+  const [reviews, setReviews] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await getReviews();
+      console.log(data);
+      setReviews(data);
+    }
+    fetchData();
+  }, [])
+
+  return (<section className="main-content">
+    <ul className="cards-list">
+      {createReviewCards(reviews)}
```

Comment:
I like your thinking however it might be one level of abstraction too far, I would just map the reviews onto reviewcards here rather than having doing it like this